Source/Models/Scheduler.swift

```diff
@@ -1,9 +1,9 @@
-public typealias Scheduler = ((() -> Void) -> Void)
+public typealias Scheduler = ((@escaping () -> Void) -> Void)
```

Comment:
I feel like this annotation isn't in the right place. The closure that gets passed _into_ the scheduler shouldn't need to be escaping, right?